terraform/nixos-rebuild/deploy.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/env -S nix shell --inputs-from ../../ nixpkgs#openssh nixpkgs#nix nixpkgs#coreutils nixpkgs#runtimeShellPackage -c bash
```

Comment:
I think this should also enable experiment flags for nix-command and flakes in case the user does not have this enabled.

also:

```suggestion
#!/usr/bin/env -S nix shell --inputs-from path:./../ nixpkgs#openssh nixpkgs#nix nixpkgs#coreutils nixpkgs#runtimeShellPackage -c bash
```

otherwise it might discover a git repository and the wrong flake.

Reply:
we had issues vendoring openssh in the past: https://github.com/numtide/nixos-anywhere/issues/62
I think openssh should be only provided as a fallback.

Reply:
one idea: `export PATH=$PATH:$(nix shell --pure ... -c sh -c 'echo $PATH')`